networking/manager.go

```diff
@@ -655,6 +655,12 @@ func validateIfConfig(ifconf *maestroSpecs.NetIfConfigPayload) (ok bool, problem
 		ok = false
 		problem = "Interface is missing IfName / if_name field"
 	}
+	if ifconf.Type == "wifi" {
```

Comment:
Now that we support multiple types, should this function check to make sure that ifconf.Type exists is one of our supported types?

Reply:
Yes, that's a good idea. But the problem is in order to configure ethernet interfaces, maestro documentation and commands do not require ifconf.Type to be set (i.e it is an empty string). Do we just return an error if ifconf.Type is not an empty string or our supported types?

Reply:
Defaulting to ethernet if the type is empty string seems fine.  Is there a central place where defaults are provided?